packages/sw-lib/src/lib/router.js

```diff
@@ -54,14 +54,13 @@ import ErrorFactory from './error-factory.js';
  * @memberof module:sw-lib
  */
 class Router extends SWRoutingRouter {
-
   /**
    * @param {String|Regex|Route} capture The capture for a route can be one
    * of three types.
-   * 1. It can be an Express style route, like: '/example/:anything/route/'
-   *    The only gotcha with this is that it will only capture URL's on your
-   *    origin.
-   * 1. A regex that will be tested against request URL's.
+   * 1. It can be an Express style route, like '/wildcard/prefix/(.*)' for
```

Comment:
Could you keep the naming here ':someName' just to probably seperate express from regex and could you include an express name in the cross-origin route.